docker-compose/.env.sample

```diff
@@ -0,0 +1,57 @@
+##############################
+### User Init Job Settings ###
+##############################
+# These variables will be used by the init jobs the first time you boot up with a fresh DB
+SUPER_ADMIN_EMAIL=
+SUPER_ADMIN_PASSWORD=
+INITIAL_REGISTRIES_JSON="eyJyZWdpc3RyaWVzIjpbeyJuYW1lIjoiRG9ja2VyaHViIiwiaG9zdG5hbWUiOiJkb2NrZXIuaW8iLCJhbGlhc2VzIjpbImluZGV4LmRvY2tlci5pbyJdLCJsb2dpbl9yZXF1aXJlZCI6ZmFsc2V9LHsibmFtZSI6IkdpdGh1YiBDb250YWluZXIgUmVnaXN0cnkiLCJob3N0bmFtZSI6ImdoY3IuaW8iLCJsb2dpbl9yZXF1aXJlZCI6ZmFsc2V9LHsibmFtZSI6Ikt1YmVybmV0ZXMgQ29udGFpbmVyIFJlZ2lzdHJ5IiwiaG9zdG5hbWUiOiJyZWdpc3RyeS5rOHMuaW8iLCJsb2dpbl9yZXF1aXJlZCI6ZmFsc2V9XX0="
```

Comment:
I think we changed this

Reply:
maybe

Reply:
We only changed how it is added to the value files.

The helm chart takes a pretty list in YAML [1], but then as part of the templating [2], it converts it to a JSON and b64 encodes that to end up with this kind of string. (This one contains the same data that you would get by using the default)

[1] https://github.com/m9sweeper/m9sweeper/blob/main/manifests/charts/dash/values.yaml#L59
[2] https://github.com/m9sweeper/m9sweeper/blob/main/manifests/charts/dash/templates/job-init.yaml#L36